docs/DeveloperGuide.adoc

```diff
@@ -253,72 +264,77 @@ image::StorageClassDiagram.png[width="800"]
 The `Storage` component,
 
 * can save `UserPref` objects in json format and read it back.
-* can save the Address Book data in json format and read it back.
+* can save the TravelBuddy data in json format and read it back.
 
 [[Design-Commons]]
 === Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `seedu.TravelBuddy.commons` package.
 
 == Implementation
 
 This section describes some noteworthy details on how certain features are implemented.
 
 // tag::undoredo[]
 === Undo/Redo feature
+
+Called by the `undo` and `redo` commands to undo and redo changes to TravelBuddy Place entries.
+Undoable commands include `add` `edit` `delete` `deletem` `clear` `generate` and 'redo'
+Redoable commands include `add` `edit` `delete` `deletem` `clear` `generate` and 'undo'
+
 ==== Current Implementation
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
+The undo/redo mechanism is facilitated by `VersionedTravelBuddy`.
+It extends `TravelBuddy` with an undo/redo history, stored internally as an `TravelBuddyStateList` and `currentStatePointer`.
```

Comment:
The correct variable name is `travelBuddyStateList`.

Reply:
Thank you for pointing that out